scripts/go.mod

```diff
@@ -0,0 +1,46 @@
+module storj.io/storj/scripts
+
+go 1.12
+
+require (
+	github.com/AlekSi/gocov-xml v0.0.0-20190121064608-3a14fb1c4737
+	github.com/akavel/rsrc v0.8.0 // indirect
+	github.com/axw/gocov v1.0.0
+	github.com/ckaznocha/protoc-gen-lint v0.2.1
+	github.com/go-bindata/go-bindata v3.1.2+incompatible
+	github.com/gogo/protobuf v1.2.1
+	github.com/golang/protobuf v1.3.2 // indirect
+	github.com/golangci/golangci-lint v1.17.1
+	github.com/josephspurrier/goversioninfo v0.0.0-20190209210621-63e6d1acd3dd
+	github.com/kylelemons/godebug v1.1.0
+	github.com/loov/leakcheck v0.0.3
+	github.com/mfridman/tparse v0.7.4
+	github.com/nilslice/protolock v0.14.0
+	github.com/zeebo/errs v1.2.2
+	golang.org/x/tools v0.0.0-20190829051458-42f498d34c4d
+	gopkg.in/spacemonkeygo/dbx.v1 v1.0.0-20190212172312-3af5e1fc2659
+)
+
+// golangci-lint has some dependencies on incorrectly published modules so we
```

Comment:
I wonder whether it would be easier to just depend on the exact binary version rther than do the replacements?

Reply:
Yeah. I mean `curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b ${GOPATH}/bin v1.17.1` downloaded the exact version without having to lock in other repository versions. The replacement thing feels fragile.

Reply:
Ok, let's leave it. But it still feels fragile :)